test/js-native-api/test_string/test_string.c

```diff
@@ -221,6 +236,14 @@ static napi_value TestUtf16External(napi_env env, napi_callback_info info) {
                          actual_length);
 }
 
+static napi_value TestPropertyKeyUtf16(napi_env env, napi_callback_info info) {
```

Comment:
The new function must be exposed to JS code. Please add it to the `napi_property_descriptor properties[] = { ...` in the end of this file.